packs/feats/emblazon-armament.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Carefully etching a sacred image into a physical object, you steel yourself for battle. You can spend 10 minutes emblazoning a symbol of your deity upon a weapon or shield. The symbol doesn't fade until 1 year has passed, but if you Emblazon an Armament, any symbol you previously emblazoned and any symbol already emblazoned on that item instantly disappears. The item becomes a religious symbol of your deity and can be used as a divine focus while emblazoned, and it gains another benefit determined by the type of item. This benefit applies only to followers of the deity the symbol represents.</p>\n<ul>\n<li><strong>Shield</strong> The shield gains a +1 status bonus to its Hardness. (This causes it to reduce more damage with the Shield Block reaction.)</li>\n<li><strong>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Emblazon Armament (Weapon)]{Weapon}</strong> The wielder gains a +1 status bonus to damage rolls.</li>\n</ul>"
+            "value": "<p>Carefully etching a sacred image into a physical object, you steel yourself for battle. You can spend 10 minutes emblazoning a symbol of your deity upon a weapon or shield. The symbol doesn't fade until 1 year has passed, but if you Emblazon an Armament, any symbol you previously emblazoned and any symbol already emblazoned on that item instantly disappears. The item becomes a religious symbol of your deity and can be used as a divine focus while emblazoned, and it gains another benefit determined by the type of item. This benefit applies only to followers of the deity the symbol represents.</p>\n<ul>\n<li><strong>@UUID[Compendium.pf2e.feat-effects.Item.o04gwnrVtzWyIEs8]{Shield}</strong> The shield gains a +1 status bonus to its Hardness. (This causes it to reduce more damage with the Shield Block reaction.)</li>\n<li><strong>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Emblazon Armament (Weapon)]{Weapon}</strong> The wielder gains a +1 status bonus to damage rolls.</li>\n</ul>"
```

Comment:
This should link to the effect item name rather than ID. It will be converted to build time.

Reply:
Extracted packs in correct order, and it now uses the name not ID